packages/core/src/utils/SwapApiUtil.ts

```diff
@@ -22,6 +26,36 @@ export type TokenInfo = {
 
 // -- Controller ---------------------------------------- //
 export const SwapApiUtil = {
+  async getQuote({
+    fromTokenAddress,
+    toTokenAddress,
+    amount,
+    accountAddress,
+    gasPrice
+  }: {
+    fromTokenAddress: string
+    toTokenAddress: string
+    amount: string
+    accountAddress: string
+    gasPrice: string
+  }) {
+    const apiUrl = new URL(`${ONEINCH_API_BASE_URL}/swap/v6.0/137/quote`)
```

Comment:
why do we need to instantiate a new api her?

Reply:
This shouldn't be here anymore

Reply:
Saw forgot to push lol